src/modules/ekf2/EKF/velocity_fusion.cpp

```diff
@@ -79,12 +82,61 @@ void Ekf::updateVelocityAidSrcStatus(const uint64_t &time_us, const Vector3f &ob
 	aid_src.timestamp_sample = time_us;
 }
 
+void Ekf::fuseBodyVelocity(estimator_aid_source3d_s &aid_src)
+{
+	Vector3f innov;
```

Comment:
No need to declare the variables on top of the function in C++, you wan just write `const Vector3f innov = _R_to_earth.transpose() * _state.vel - Vector3f(aid_src.observation);`